libraryValidations/Python/test_json_validations.py

```diff
@@ -109,25 +132,49 @@ def run_tests(self, test_key):
                 groups = feature_flag_test[INPUTS_KEY].get(GROUPS_KEY, [])
                 assert (
                     feature_manager.is_enabled(
-                        feature_flag_test[FEATURE_FLAG_NAME_KEY], TargetingContext(user_id=user, groups=groups)
+                        feature_flag_test[FEATURE_FLAG_NAME_KEY],
+                        TargetingContext(user_id=user, groups=groups),
                     )
                     == expected_is_enabled_result
                 ), failed_description
             else:
                 with raises(ValueError) as ex_info:
-                    feature_manager.is_enabled(feature_flag_test[FEATURE_FLAG_NAME_KEY])
+                    feature_manager.is_enabled(
+                        feature_flag_test[FEATURE_FLAG_NAME_KEY],
+                        TargetingContext(user_id=user, groups=groups),
+                    )
                 expected_message = is_enabled.get(EXCEPTION_KEY)
                 assert str(ex_info.value) == expected_message, failed_description
 
-            if get_variant is not None and RESULT_KEY in get_variant:
+            if get_variant and (
```

Comment:
I think this check should just be `if get_variant`

And we'll conditionally try the exception or result within it. Worst case, we run the function when the json has {} or a typo in that section.